log4cl-examples.asd

```diff
@@ -13,12 +13,8 @@
 ;;; See the License for the specific language governing permissions and
 ;;; limitations under the License.
 
-(asdf:defsystem #:log4cl-examples
-  :version "1.1.2"
-  :depends-on (#:log4cl #:swank)
-  :components ((:file "examples/naming-examples") 
+(defsystem "log4cl-examples"
+  :version (:read-file-form "version-string.sexp")
+  :depends-on ("log4cl" "swank")
+  :components ((:file "examples/naming-examples")
                (:file "examples/customize-log-expr")))
-
```

Comment:
are we sure we want to break clisp right of the bat? `:read-file-form` is not backward compatible (maybe `#. …` magic?). also having asd files without any package (neither being in asdf-user nor having `asdf:defsystem` explicit accessor) makes C-c C-k painful during development - because it will fail.

Reply:
Note, we still depend on BT which doesn't work there either way.